internal/witness/verifier/verifier.go

```diff
@@ -49,11 +49,18 @@ func (wv WitnessVerifier) VerifySignature(sth api.CosignedSTH) error {
 	if err != nil {
 		return fmt.Errorf("failed to marshal internal STH: %v", err)
 	}
-	for i, sig := range sth.WitnessSigs {
+	oneVerifies := false
+	for _, sig := range sth.WitnessSigs {
```

Comment:
I think you can slightly simplify this by doing:

```golang
for _, sig := range sigs {
  if verify(data, sig) {
    return nil
  }
}

return "oh noes"
```
?

Reply:
Yes, very good point :).